src/webview-bridge-cjs.js.flow

```diff
@@ -65,6 +65,8 @@ declare export function requestSimImsi(): Promise<string | null>;
 
 declare export function requestDeviceImei(): Promise<string | null>;
 
+declare export function internalNavigation(feature: string): Promise<void>;
```

Comment:
type param with available features:
```js
internalNavigation(feature: 'internal-navigation')
```

Reply:
I've talked with Yamal and I see that right now it only support 'notification-settings' as feature, so I'll change this